toqito/channel_ops/choi_to_kraus.py

```diff
@@ -3,7 +3,7 @@
 
 import numpy as np
 
-from toqito.helper import channel_dim
+from toqito.channel_props import channel_dim
 from toqito.matrix_ops import unvec
 from toqito.matrix_props import is_hermitian, is_positive_semidefinite
 
```

Comment:
There is a circular import issue related to this function. 

In https://github.com/vprusso/toqito/pull/262/commits/4cd69fd1c08fab978c69ed2912b42035936827a1, I had to specify the full path to try to fix the failures. 

But the workflow run is going to fail due to a test failure in `is_unitary`. In this case, the failure is due to ` TypeError: 'module' object is not callable` for `choi_to_kraus`. 

If I try to specify the full path at the line linked below similar to https://github.com/vprusso/toqito/pull/262/commits/4cd69fd1c08fab978c69ed2912b42035936827a1, the tests fail
due to a circular import error. 

https://github.com/vprusso/toqito/blob/a3035078e63e727b4a3eae963c30da7a76ba00ba/toqito/channel_props/is_unitary.py#L6

```
ImportError: cannot import name 'choi_to_kraus' from partially initialized module 'toqito.channel_ops.choi_to_kraus' (most likely due to a circular import)
```

Reply:
Fixing this will need a bigger refactor. There is now an issue for this.